walkin/tests/test_management_commands.py

```diff
@@ -0,0 +1,134 @@
+from datetime import timedelta
+
+from django.core.management import call_command
+from django.utils import timezone
+
+from opal.core.test import OpalTestCase
+from elcid.models import PresentingComplaint
+from walkin.models import Symptom
+
+
+class SymptomToPresentingComplaintTest(OpalTestCase):
+
+    def setUp(self):
+        self.patient, self.episode = self.new_patient_and_episode_please()
+        self.symptom1 = Symptom.objects.create(
+            id=1,
+            created=           timezone.now(),
+            updated=           timezone.now(),
+            created_by=        self.user,
+            updated_by=        self.user,
+            consistency_token= u'charfield 24167234',
+            episode=           self.episode,
+            duration=          u'charfield some duration',
+            details=           u'some details',
+            onset=             u'charfield onset',
+            symptom=           u'ftfk symptom',
+        )
+        self.symptom1.save()
+        self.symptom2 = Symptom.objects.create(
+            id=2,
+            created=           timezone.now()-timedelta(1),
+            updated=           timezone.now()-timedelta(1),
+            created_by=        self.user,
+            updated_by=        self.user,
+            consistency_token= u'charfield 16723461',
+            episode=           self.episode,
+            duration=          u'charfield some other duration',
+            details=           u'some other details',
+            onset=             u'charfield some onset',
+            symptom=           u'ftfk a different symptom',
+        )
+        self.symptom2.save()
+
+    def test_there_are_no_symptom_obejcts_to_migrate(self):
+        self.symptom1.delete()
+        self.symptom2.delete()
+        call_command('move_symptoms_to_presentingcomplaints')
+        pc1 = PresentingComplaint.objects.first()
```

Comment:
Conceptually here you're tying your tests to the implementation working in a particular order - which is  in fact not the thing you want to tie them to... 

You might consider `pc1 = PresentingComplaint.objects.get(details='some details')` as an alternative that won't break all your tests if you decide to iterate through symptoms in reverse.

Same thing with get by id=integer below.

This is probably fine for these, but in general worth avoiding... 

If your tests implicitly rely on implementation details that aren't really the thing you're testing this can be frustrating when they break later on despite the functionality you're testing still working :)